server/info.go

```diff
@@ -636,6 +636,13 @@ func (s *infoServer) mergeAdditionalInfoToTxs(txs []*types.Transaction, receipts
 		}
 
 		tx.Logs = receipts[receiptIndex].Logs
+		for i, log := range tx.Logs {
+			result, err := s.kaiClient.UnpackLog(&log)
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>G601: Implicit memory aliasing in for loop. (gosec)

Reply:
Resolved